src/components/count/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import { chakra, useRecipe } from '@chakra-ui/react';
+
+export type CountVariant =
+  | 'primary'
+  | 'inverted'
+  | 'info'
+  | 'success'
+  | 'warning'
+  | 'error';
+
+export type CountProps = {
+  count: number;
+  variant?: CountVariant;
```

Comment:
Since this is already declared in the recipe and the types are inferred from recipes you don't need to re-declare the variants here. Check out the [example in the docs](https://www.chakra-ui.com/docs/theming/recipes#typescript)